monai/data/samplers.py

```diff
@@ -95,15 +95,14 @@ def __init__(
         even_divisible: bool = True,
         num_replicas: int | None = None,
         rank: int | None = None,
-        shuffle: bool = True,
         **kwargs,
     ):
+        kwargs.setdefault("shuffle", True)
```

Comment:
thanks! please update the docstring and check the test case https://github.com/Project-MONAI/MONAI/blob/4cd717ca830ed7cd811144d96675fdbe586633a4/tests/test_weighted_random_sampler_dist.py#L26

Reply:
Checked the test case:
![image](https://github.com/Project-MONAI/MONAI/assets/84932095/edece67d-f0c1-4015-b418-21f4139674df)